Discord.py

```diff
@@ -83,11 +83,25 @@
                     output = "No results found for " + ckey
 
         finally:
-            await client.send_message(message.channel, output)
+            trimmed_output = await discord_msg_trim(output)
+            for message in trimmed_output:
+                await client.send_message(message.channel, message)
             del(cursor)
             db.close()
 
 
+async def discord_msg_trim(string):

```

Comment:
Doing it this way could seriously screw with the formatting. I'd rather that around lines 78-81 you check whether the current note would push the message over 2,000 characters and flush it out to Discord; that would prevent formatting from breaking.

Reply:
I just thought of that actually and made a quick change to actually format the messages within this function itself so each message is encapsulated within a triple `.